book/website/collaboration/research-infrastructure-roles/research-software-engineering/research-software-engineering-casestudy.md

```diff
@@ -0,0 +1,113 @@
+(cl-infrastructure-rse-casestudy)=
+# Research Software Engineering Case Study
```

Comment:
```suggestion
# Research Software Engineering Personal Story
```

I think this chapter and file should be renamed to 'personal stories', instead of case study. RSE set up around the world would look different, hence we can feature multiple personal stories starting from this one.